### ISSUE TITLE
ccRides crashes on startup

### ISSUE THREAD

**Issue by dwightdurmon** [TheClowner/ccRides#1]
**Snippet from Logs**
```
06:22:37 INFO]: [ccRides] Enabling ccRides v1.0.0
[06:22:37 INFO]: [ccRides] Main Configs Loaded
[06:22:37 INFO]: [ccRides] No Rides Configured yet.
[06:22:37 ERROR]: Error occurred while enabling ccRides v1.0.0 (Is it up to date?)
java.lang.NullPointerException: null
at java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006) ~[?:?]
at java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011) ~[?:?]
at net.clownercraft.ccRides.RidesPlugin.onEnable(RidesPlugin.java:52) ~[?:?]
at net.clownercraft.ccRides.config.ConfigHandler.<init>(ConfigHandler.java:100) ~[?:?]
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:472) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:386) ~[patched_1.15.2.jar:git-Paper-375]
at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:488) ~[patched_1.15.2.jar:git-Paper-375]
at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:908) ~[patched_1.15.2.jar:git-Paper-375]
at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:298) ~[patched_1.15.2.jar:git-Paper-375]
at java.lang.Thread.run(Thread.java:834) [?:?]
[06:22:37 ERROR]: Error occurred while disabling ccRides v1.0.0 (Is it up to date?)
[06:22:37 INFO]: [ccRides] Disabling ccRides v1.0.0
at net.clownercraft.ccRides.RidesPlugin.onDisable(RidesPlugin.java:137) ~[?:?]
java.lang.NullPointerException: null
at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:415) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:516) ~[patched_1.15.2.jar:git-Paper-375] at net.clownercraft.ccRides.RidesPlugin.onDisable(RidesPlugin.java:137) ~[?:?](B
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:384) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:415) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:516) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:472) ~[patched_1.15.2.jar:git-Paper-375]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:386) ~[patched_1.15.2.jar:git-Paper-375]
at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:488) ~[patched_1.15.2.jar:git-Paper-375]
at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:908) ~[patched_1.15.2.jar:git-Paper-375]
at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:298) ~[patched_1.15.2.jar:git-Paper-375]
at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Steps to reproduce:**
1. Upload plugin to server
2. Restart server
3. Wait for crash message

version
```
[06:26:28 INFO]: Checking version, please wait...
>> = [06:26:28 INFO]: This server is running Paper version git-Paper-375 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[06:26:28 INFO]: You are 3 version(s) behind

```
**plugins**

```
>> = [06:29:11 INFO]: Plugins (63): AdvancedEnchantments, Alchemist*, AnimationLib, AsyncWorldEdit, BCommons*, BetterFarming, BigDoors, BigDoorsOpener, Blacksmith, Builder*, ccRides, ChestSort, Citizens, CleanroomGenerator*, CraftBook, CustomGUI, DeluxeMenus, Denizen, DisplayShops, DivineDrop*, DoorCloser, dtlTradersPlus, DynamicLights, Essentials, EssentialsChat, EssentialsSpawn, FastCraft, FastLeafDecay, GadgetsMenu, GriefPrevention, Harbor, HolographicDisplays, HolographicExtension, InfiniteScoreboard, ItemJoin, Jobs, JoinCommands*, LibsDisguises, LightAPI, LuckPerms, MovingDevApi, Multiverse-Core, Multiverse-Inventories, MythicMobs, NPC_Destinations, NPCDestinations_Animator, NPCDestinations_Farmer, OwnGarden, PlaceholderAPI, PortableCraftingInvs, ProtocolLib, Sentinel, ServerTutorialPlus*, TheNewEconomy, TimedRewards*, TradePlus, UltimateTimber, Vault, Vehicles, VoidGenerator, WarpSystem, WorldEdit, WorldGuard
```

Let me know if I can add any other details.

**Comment by dwightdurmon** [comment]
**Steps to reproduce:**
1. Upload plugin to server
2. Restart server
3. Wait for crash message

`version
[06:26:28 INFO]: Checking version, please wait...
>> = [06:26:28 INFO]: This server is running Paper version git-Paper-375 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[06:26:28 INFO]: You are 3 version(s) behind
`

`plugins
>> = [06:29:11 INFO]: Plugins (63): AdvancedEnchantments, Alchemist*, AnimationLib, AsyncWorldEdit, BCommons*, BetterFarming, BigDoors, BigDoorsOpener, Blacksmith, Builder*, ccRides, ChestSort, Citizens, CleanroomGenerator*, CraftBook, CustomGUI, DeluxeMenus, Denizen, DisplayShops, DivineDrop*, DoorCloser, dtlTradersPlus, DynamicLights, Essentials, EssentialsChat, EssentialsSpawn, FastCraft, FastLeafDecay, GadgetsMenu, GriefPrevention, Harbor, HolographicDisplays, HolographicExtension, InfiniteScoreboard, ItemJoin, Jobs, JoinCommands*, LibsDisguises, LightAPI, LuckPerms, MovingDevApi, Multiverse-Core, Multiverse-Inventories, MythicMobs, NPC_Destinations, NPCDestinations_Animator, NPCDestinations_Farmer, OwnGarden, PlaceholderAPI, PortableCraftingInvs, ProtocolLib, Sentinel, ServerTutorialPlus*, TheNewEconomy, TimedRewards*, TradePlus, UltimateTimber, Vault, Vehicles, VoidGenerator, WarpSystem, WorldEdit, WorldGuard`

**Comment by TheClowner** [comment]
This should be fixed in v1.0.2 - which is now available on spigot
